### PR TITLE
(DB) Spanish translation (elwynn forest) "quest_offer_reward_locale" and "quest_request_items_locale".

### PR DESCRIPTION
**Pull request by Le-Math**
## CHANGES PROPOSED:
-  Translate several texts from "quest_offer_reward_locale" and "quest_request_items_locale" into Spanish.

## ISSUES ADDRESSED:
- Closes 

## TESTS PERFORMED:
-Sql imported and own tests performed.

## HOW TO TEST THE CHANGES:

-Importing them directly to world

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Helias** [comment]
add `DELETE` before `INSERT`, moreover, you can group the `INSERT INTO` in the following way

before:
```sql
insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('783','esES','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.','18019');
insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('783','esMX','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.','18019');

insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('5261','esES','Es verdad. Estoy buscando a alguien para cazarme lobos! ¿Acaso eres esa persona?','18019');
insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('5261','esMX','Es verdad. Estoy buscando a alguien para cazarme lobos! ¿Acaso eres esa persona?','18019');
```

after:
```sql
DELETE FROM `quest_offer_reward_locale` WHERE `ID` IN (783, 5261) AND `locale` = 'esES';
INSERT INTO `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) VALUES
(783,'esES','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.',18019),
(5261,'esES','Es verdad. Estoy buscando a alguien para cazarme lobos! ¿Acaso eres esa persona?', 18019);


DELETE FROM `quest_offer_reward_locale` WHERE `ID` IN (783, 5261) AND `locale` = 'esMx';
INSERT INTO `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) VALUES
(783,'esMX','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.', 18019),
(5261,'esMX','Es verdad. Estoy buscando a alguien para cazarme lobos! ¿Acaso eres esa persona?', 18019);
```

NOTE: I also converted string like '783' into INTEGERS, do the same pls.

File: data/sql/updates/pending_db_world/rev_1607503600897995800.sql [hunk 1]
```diff
@@ -0,0 +1,257 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1607503600897995800');
+insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('783','esES','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.','18019');
+insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('783','esMX','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.','18019');
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1607503600897995800.sql:3]: - convert string with numbers into integers
- regroup the INSERT INTO
- add DELETE FROM before insert

**Review by Helias — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1607503600897995800.sql (hunk 1 above)
Comment (Le-Math) [reply data/sql/updates/pending_db_world/rev_1607503600897995800.sql:3]: Thank you very much, I didn't know.
Next time I will do it :)

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1607503600897995800.sql:3]: it's better to do it in this PR, you can easily edit the PR even it is opened.

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1607503600897995800.sql]: I see your edits, convert the string like `'793'` into integer like `793` without quotes and put a newline between the VALUES for a better visualization pls
just this

File: data/sql/updates/pending_db_world/rev_1607503600897995800.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
...estino de Rolf y Malakai?','18019'),('59','esMX','He sido comisionada por el ejército de ventormenta para proveer a su gente armaduras de tela y cuero.$B$BSi tienes material para mí, entonces estaría encantada de hacer algo por ti.','18019'),('46','esMX','¿Cómo va la cacería $N?','18019'),('106','esMX','¿Tienes qué? Maybell es la luz de mi aburrida vida. ¡Rápido, déjame ver su carta!','18019'),('107','esMX','Tienes algo de "Abuela" Pedregosa, ¿eh? ¡No he visto a Mildred en años! Me pregunto qué tiene que decir...','18019'),('114','esMX','¿Le entregaste mi carta a Tommy Joe? ¿Qué dijo?','18019'),('88','esMX','¿La has visto ya? ¿La has visto?','18019'),('86','esMX','No creo que esté bien alimentar al chico que robó mi collar en primer lugar, pero si eso es lo que se necesita para recuperar lo que es mío ¡entonces que así sea!$B$B¿Tienes carne de jabalí?','18019'),('84','esMX','¡Ugh, me muero de hambre! ¿Ya tienes ese pastel para mí, $N?','18019'),('87','esMX','Hola $N. ¿Has encontrado mi collar?','18019'),('16','esMX','Cultivar es un trabajo sediento, y siempre estoy buscando agua de mamantial refrescante.$B$BSi tienes agua, entonces estoy dispuesto a hacer un intercambio.','18019');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1607503600897995800.sql:13]: 1. Remove quotes from numbers. They are ints not strings.
2. Keep everything on new lines like this
DELETE XXX
INSERT XXX VALUES
(X, x, x),
(X, x, x);

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1607503600897995800.sql (hunk 2 above)
Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1607503600897995800.sql]: I did it, check it ;-)

**Review by Helias — APPROVED** [review]

**Review by Kitzunu — APPROVED** [review]
Can run it twice, good for me

**Comment by Kitzunu** [comment]
ty =)

**Comment by MathematicalWTF** [comment]
Thank you for correcting it.
I would have done it myself, but I lost my password at the time.

**Pull request merged**